bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -321,8 +371,12 @@ export namespace DataTable {
   export type Attrs = p.AttrsOf<Props>
 
   export type Props = TableWidget.Props & {
+    autosize_mode: p.Property<string>
```

Comment:
Use `"fit_columns" | "fit_viewport" ...` instead of `string`. Then you won't need to use `any` casting in a few places.

Reply:
Thanks